tests/integration/pilot/testdata/serviceEntry.yaml

```diff
@@ -0,0 +1,1017 @@
+apiVersion: v1
+kind: Node
```

Comment:
A couple of things...

1) I think you'll want to split apart the resources into separate files for clarity
2) You'll want to apply different files in different environments.  For example, in k8s, you don't apply Nodes, Pods, or Endpoints ... you only apply service and deployment.